### ISSUE TITLE
Support PHPUnit\Framework\TestCase::setUp in missing class property inspection

### ISSUE THREAD

**Issue by Haehnchen** [kalessil/phpinspectionsea#812]
PhpUnit plugin is supporting properties via setUp (extending constructor functionality here), as this is a common pattern for unit tests. see https://github.com/Haehnchen/idea-php-phpunit-plugin#mocks. But the property is highlight in this plugin. It would be nice to support this pattern here also.

Just a hint: As PhpUnit  plugin attach the types to the property it would an idea to reuse this PhpStorm functionally. If type is known to PhpStorm dont provide an error. It looks like you are scanning manually for usages.

```php
class FoobarTest extends TestCase
{
    protected function setUp()
    {
         $this->foobar = $this->createMock(Foobar::class);
    }

    publuc function testFoo()
    {
        // missing property detection fired
        $this->foobar->foobar();
    }
}
```

**Comment by rentalhost** [comment]
I think you should declare this property with the correct `@type`. Eg.:

```php
/** @type MockObject */
private $foobar;
```

**Comment by kalessil** [comment]
Hi, Daniel =) Will check if we are emitting the case.

**Comment by kalessil** [comment]
@rentalhost : here something weird happens, I'll take care as it's on the stack of IDE and 2 3rd party plugins.

**Comment by kalessil** [comment]
@Haehnchen : checked, the warning is by PhpUndefinedFieldInspection (IDE bundled one). Worth reporting to PhpStrom team IMO.
